source/frontend/src/features/mapSideBar/acquisition/common/GenerateForm/hooks/useGenerateExpropriationForm1.test.tsx

```diff
@@ -131,16 +131,13 @@ describe('useGenerateExpropriationForm1 functions', () => {
   it(`sends "Expropriation notice served date" in the json payload to generation api`, async () => {
```

Comment:
this test is explicitly about the expropriation date field you removed, so the test itself should be removed as well.